### PR TITLE
Redirect for old vet verification auth docs to the new auth docs

### PR DESCRIPTION
**Pull request by ediiotero**
Story: [API-4636](https://vajira.max.gov/browse/API-4636)

### Redirect old vet verification auth docs

Adding a redirect from old auth docs to the new docs with vet verification preselected.

### CODE REVIEW

**Review by mjlumetta — DISMISSED** [review]
looks great!! only suggestion I'd make would be to consider adding an integration test (which would be skipped at present like the others are, presumably, but could be useful to include).

**Review by gileswells — DISMISSED** [review]
Wasn't where I expected the redirect to go but I like it better here.

File: src/containers/documentation/DocumentationRoot.tsx [hunk 1]
```diff
@@ -135,9 +135,8 @@ const DocumentationRoot = (): JSX.Element => {
           {oldRouteToNew.map(routes => (
             <Redirect key={routes.from} exact from={routes.from} to={routes.to} />
           ))}
-          {authDocsV2 && (
-            <Route path="/explore/authorization" component={AuthorizationDocs} exact />
-          )}
+          {authDocsV2 && <Route path="/explore/authorization" component={AuthorizationDocs} exact />}
+          {authDocsV2 && <Redirect exact from="/explore/verification/docs/authorization" to="/explore/authorization?api=veteran_verification" />}
```
Comment (mjlumetta) [inline src/containers/documentation/DocumentationRoot.tsx:139]: any particular reason for making these blocks separate this time?

**Review by mjlumetta — APPROVED** [review]

File: src/containers/documentation/DocumentationRoot.tsx (hunk 1 above)
Comment (ediiotero) [reply src/containers/documentation/DocumentationRoot.tsx:139]: The previous way I had it set up caused the other authDocsV2 redirect to break, not sure why. Separated them and all the routes worked as expected and the tests passed.

Comment (mjlumetta) [reply src/containers/documentation/DocumentationRoot.tsx:139]: hmm... I'm pretty confused why that would've happened - they should be functionally equivalent.

Comment (ediiotero) [reply src/containers/documentation/DocumentationRoot.tsx:139]: yeah same here

**Pull request merged**